server/controllers/payroll/multiplePayroll/find.js

```diff
@@ -39,7 +39,7 @@ function find(options) {
         paiement.payroll_configuration_id,  paiement.currency_id, paiement.paiement_date, paiement.base_taxable,
```

Comment:
Since we are using English for variable names, I suggest changing 'paiement' to 'payment' where practical.

Reply:
Wow!  Glad you did that globally!  Thanks